spec/features/__snapshots__/has-consistent-output-with-rspec-retry-test-results-rspec-3.8.gemfile.snap

```diff
@@ -10,7 +10,7 @@
     "otherErrors": 0,
     "pended": 3,
     "quarantined": 0,
-    "retries": 0,
```

Comment:
There is a bug in ABQ we should fix too:
https://github.com/rwx-research/abq/issues/497

Reply:
Talked about this over in the ABQ issue.

TL;DR it's not a bug, it's just poorly named (my bad) https://github.com/rwx-research/test-results-schema/blob/badaa101c083d595fbf86692d7cf7ae34b0c4012/v1/summary.json#L57.